core-web/libs/block-editor/src/lib/shared/components/suggestion-list/components/suggestions-list-item/suggestions-list-item.component.ts

```diff
@@ -28,7 +29,9 @@ export class SuggestionsListItemComponent implements FocusableOption, OnInit {
     @HostListener('mousedown', ['$event'])
     onMouseDown(e: MouseEvent) {
         e.preventDefault();
-        this.command();
+        if (!this.disabled) {
+            this.command();
+        }
```

Comment:
Why do we need this?

Reply:
to ignore the command execution when the option is disabled.